### ISSUE TITLE
[COMPONENT] Text component

### ISSUE THREAD

**Issue by bommariusser** [kirbydesign/designsystem#1352]
**Describe the solution you'd like**
We would like a Text component which has some built-in padding on it, and would have the same responsive behavior like some of our other components like Cards and Tabbar. 

This padding is most of the time 16px left/right, but we've thought about taking this to 8px on smaller screens, like we do with our page padding. 

![Skærmbillede 2021-02-24 kl  12 38 03](https://user-images.githubusercontent.com/49033002/108996472-b5af3180-769e-11eb-880a-8bb975a4f976.png)

<img width="403" alt="Skærmbillede 2021-03-23 kl  10 09 13" src="https://user-images.githubusercontent.com/71710012/112121731-d98f6580-8bbf-11eb-808b-8e02d043051a.png">

**Describe alternatives you've considered**
We have first thought of putting the padding on the page but this would come with some disadvantages...

**Additional context**
The built-in padding should be responsive for all breakpoints like our page.

### Kick Off:

- [ ] Ensure Feature is `UX refined` and aligned with UX incl. name of component (our shared Design System Language).  
_The component should be published and available in the [Kirby Styleguide on Zeplin](https://zpl.io/258pXGj)_
- [ ] Ensure Feature has been `Tech refined` with @kirbydesign/kirby-guild and this issue is updated with a clear implementation description  
_This issue should be in the [Ready to do](https://github.com/kirbydesign/designsystem/projects/1#column-4590936) column of the [Kirby kan-ban board](https://github.com/kirbydesign/designsystem/projects/1) before starting implementation)_
- [ ] Assign yourself to this issue and move it to the [In progress](https://github.com/kirbydesign/designsystem/projects/1#column-4590937) column of the [Kirby kan-ban board](https://github.com/kirbydesign/designsystem/projects/1)

### Code:

- [ ] Create Feature Branch from [master branch](https://github.com/kirbydesign/designsystem/tree/master)
- [ ] Implement Skeleton Component and push to Github
- [ ] Ask a member of @kirbydesign/kirby-guild for a WIP review by creating a draft Pull Request
- [ ] Implement unit tests
- [ ] Implement Cookbook Examples and Showcase, i.e. see [Radio](https://cookbook.kirby.design/home/showcase/radio)  
_Also remember to add any relevant API documentation_

### Review:

- UX review:
  - [ ] Ensure implementation is correct in relation to the UX design and the [Kirby Styleguide on Zeplin](https://zpl.io/258pXGj)
  - [ ] With UX agree on the version of the implementation + any additional steps needed to reach version 1.0 (create [enhancement requests](https://github.com/kirbydesign/designsystem/issues/new/choose))
- Code review:
  - [ ] Open a pull request (or mark the existing draft PR as `Ready for review`) and ask @kirbydesign/kirby-guild for a review  
_Remember to add `closes #issueno` to the description of the PR._
  - [ ] Once approved, merge feature branch/PR to master
- [ ] Ask a member of @kirbydesign/kirby-guild to add a link to component showcase from Kirby Component Status

:tada: Celebrate

**Comment by henrikvoetmand** [comment]
Added page content illustration.

**Comment by jkaltoft** [comment]
**Tech refinement**

Create a layout construct that can be wrapped around content on which there should be padding/margin in the inline dimension (left and right in LTR languages). Content that can be wrapped is not exclusively text content.

By *not* using that layout construct one can let content break out and touch both sides of the container, e.g. the Card component.

**Comment by jkaltoft** [comment]
Related to #1551 

Can be solved independently, but consider solving in tandem.

**Comment by RasmusKjeldgaard** [comment]
Consider naming it Kirby Page Section, because it can be used for many different things other than text. 
Kirby Page Section is in line with existing elements used on page, e.g. Kirby Page Header, Footer, etc.

**Comment by RasmusKjeldgaard** [comment]
**Refinement:** 
Remember to update `Page` example in cookbook with the new component (wrap existing text in the component), and possibly also introduce a card on the examples, in between some of the sections, to demo different types of layout containers.

**Comment by jkaltoft** [comment]
I think we should avoid using the name `section` because a HTML `<section>` is not a generic container and should not be used as target for styling purposes.

> The section element is not a generic container element. When an element is needed only for styling purposes or as a convenience for scripting, authors are encouraged to use the div element instead. A general rule is that the section element is appropriate only if the element's contents would be listed explicitly in the document's outline.
> https://html.spec.whatwg.org/multipage/sections.html#the-section-element

Having a `<kirby-section>` component gives the impression that it's a "componentized" HTML `<section>`.

Other than that I also think it's best to leave block axis margin on `<p>` elements out of scope for this issue.

**Comment by jkaltoft** [comment]
We could consider using a similar approach to this as Ionic: https://ionicframework.com/docs/layout/css-utilities#content-space
